lxc/list.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/gosexy/gettext"
 	"github.com/lxc/lxd"
+//	"github.com/olekukonko/tablewriter"
```

Comment:
I somehow missed that in review (not fully awake yet I guess) but that's the module we probably want to use for list in the near future, though indeed didn't have to be in there :)